tests/test_sanitize.py

```diff
@@ -37,6 +37,9 @@ def test_sanitize():
         assert_raises(CorruptedMorphology, sanitize,
                       Path(PATH, 'negative-diameters.asc'), Path(tmp_folder, 'negative-diameter.asc'))
 
+        assert_raises(CorruptedMorphology, sanitize,
+                      Path(PATH, 'neurite-with-multiple-types.swc'), Path(tmp_folder, 'bla.swc'))
+
```

Comment:
These 3 assert_raises could be improved to check that the CorruptedMorphology exception is raised for the reason that we expect. I think that checking a substring in the exception message can be enough.